sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -105,4 +104,5 @@ export enum AppRoutes {
   AssessmentSummary = "assessment-summary",
   ApplicationException = "application-exceptions/:exceptionId",
   StudentAppealRequest = "request-changes/:appealId",
+  NoticeOfAssessmentView = "notice-of-assessment/:assessmentId",
```

Comment:
Should it be `assessment/:assessmentId/notice-of-assessment`?

Reply:
ya, it makes sense, since it was already there, I didn't change it.  Updated